lib/CXGN/Trial/ParseUpload/Plugin/MultipleTrialDesignGeneric.pm

```diff
@@ -56,7 +56,10 @@ sub _validate_with_plugin {
     my $parser = CXGN::File::Parse->new(
         file => $filename,
         required_columns => \@REQUIRED_COLUMNS,
-        optional_columns => \@OPTIONAL_COLUMNS
+        optional_columns => \@OPTIONAL_COLUMNS,
+        column_aliases => {
+            'accession_name' => [ 'stock_name', 'cross_name', 'cross_unique_id' ]
```

Comment:
Please avoid using the term "cross_name". It might sound trivia. However, this term created a lot of problem in the past. At the beginning, we used the term 'cross name' as a unique identifier. For plant breeders, they often use cross combination (parentA x parentB) and call it as a cross name. That's why we changed to 'cross_unique_id'.

Reply:
updated to remove `cross_name` as a possible column alias.